cmd/run.go

```diff
@@ -58,6 +58,10 @@ func SetupRunFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().String("postgres-uri", "", "BETA: Use Postgres as the backing store for chain data."+
 		"When enabled, most data is stored in postgres although badger is still currently used for some state. Run your "+
 		"Postgres instance on the same machine as your node for optimal performance.")
+	cmd.PersistentFlags().String("pos-validator-seed", "", "The private key of the Proof of Stake validator. "+
+		"The private key should be passed as hex, optionally prefixed with a '0x', and map to a valid BLS12_381 private key. "+
```

Comment:
Do we want a private key hex or a seed phrase? We should just be clear about how we want this passed in. should we call this `pos-validator-seed-hex` if we want a hex?

Reply:
Sure. Adding the "hex" suffix makes it clearer

Reply:
Agreed!!